test/uk/gov/hmrc/traderservices/model/ImportQuestionsFormatSpec.scala

```diff
@@ -26,11 +26,10 @@ class ImportQuestionsFormatSpec extends UnitSpec {
 
     "serialize and deserialize ImportRequestType" in new JsonFormatTest[ImportRequestType](info) {
 
-      ImportRequestType.values.size shouldBe 3
```

Comment:
Very minor (and totally optional) but for consistency I think it might be nice to replicate this check for the export questions on ExportQuestionsFormatSpec?